tests/utils/test_resource_utils.py

```
@@ -0,0 +1,306 @@
+        result = create_embedded_resource(
+            "test.txt", "Hello, world!", "text/plain", False
+        )
+
+        assert isinstance(result, EmbeddedResource)
+        assert result.type == "resource"
+        assert isinstance(result.resource, TextResourceContents)
+        assert result.resource.uri == AnyUrl(url="resource://fast-agent/test.txt")
+        assert result.resource.mimeType == "text/plain"
+        assert result.resource.text == "Hello, world!"
+
+    def test_create_embedded_resource_binary(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)